icu4c/source/test/fuzzer/break_iterator_fuzzer.cpp

```diff
@@ -1,6 +1,7 @@
 // © 2019 and later: Unicode, Inc. and others.
 // License & terms of use: http://www.unicode.org/copyright.html
 
+#include <cstring>
 #include <stddef.h>
 #include <stdint.h>
 #include <string.h>
```

Comment:
this file (for example) does NOT seem properly protected against `UCONFIG_NO_BREAK_ITERATION=1`. 

This may be a good reason to make the fuzzers OFF by default unless they are requested to be turned on.

Reply:
Will review the fuzzer targets accordingly.

Reply:
I agree with the approach of turning the fuzzer build off by default, and not bothering with UCONFIG ifdefs in the fuzzer code.

Reply:
Agree.